playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml

```diff
@@ -21,15 +21,23 @@
   gather_facts: false
   connection: local
   tasks:
+
+    - name: Remove ssh key entry
```

Comment:
so, we shouldn't need this. As our playbook adds that info on the controller side.

Can you setup an autohold on failure so I can look.  I suspect something in the controller is regenerating the host keys for some reason